.github/workflows/ci.yml

```diff
@@ -198,21 +204,21 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
-          cache-to: type=gha,mode=max  
-      
+          cache-to: type=gha,mode=max
+
       - name: Notify if Job Fails
         uses: ravsamhq/notify-slack-action@v1
         if: always()
         with:
           status: ${{ job.status }}
           token: ${{ secrets.GITHUB_TOKEN }}
-          notification_title: '{workflow} has {status_message}'
-          message_format: '{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}> : <{run_url}|View Run Results>'
-          footer: ''
-          notify_when: 'failure'
+          notification_title: "{workflow} has {status_message}"
```

Comment:
As tempting as it is to drive-by fix formatting, do this in a separate PR.

Reply:
Seems like VS Code formatted it based on YAML rules, is it okay to leave or would you like me to still revert?

Reply:
Rather than reverting, make a new PR against `master` that performs the formatting, merge that in, then merge `master` into this branch. Boom, now all your formatting changes just get cancelled out for free!

Reply:
https://github.com/FuelLabs/fuel-core/pull/360

Reply:
https://github.com/FuelLabs/fuel-core/pull/360